tests/php/test-class-amp-meta-sanitizer.php

```diff
@@ -30,7 +61,42 @@ public function get_data_for_sanitize() {
 
 		$amp_boilerplate = amp_get_boilerplate_code();
 
-		return [
+		$meta_charset  = '<meta charset="utf-8">';
```

Comment:
These tests here are much more extensive than what I had before and covers the happy path of execution, along with potential mishaps that may occur :+1:.